tools/sof-kernel-log-check.sh

```diff
@@ -345,6 +345,15 @@ ignore_str="$ignore_str"'|iwlwifi [[:digit:].:]+: '
 # SDW related logs
 #
 
+#Irrelated log brought in verify-kernel-boot-log.sh
+#http://sof-ci.sh.intel.com/#/result/planresultdetail/4120?model=CML_RVP_HDA&testcase=verify-kernel-boot-log
+ignore_str="$ignore_str"'|Failed to mount NFSD configuration filesystem'
+#http://sof-ci.sh.intel.com/#/result/planresultdetail/3937?model=TGLH_0A70_HDA&testcase=verify-kernel-boot-log
+ignore_str="$ignore_str"'|DMAR: DRHD: handling fault status reg '
```

Comment:
@kv2019i any comment on this one? I googled it and it seems to be related to some IOMMU (mis)configuration in ACPI... confirm? Any workaround?

Reply:
@marc-hb Sorry, missed the question, so bit of a delay. This is most probably a BIOS issue. I think it is safe to ignore especially in the initial boot. Otherwise we'd need to disable IOMMU, but that's explicitly enabled in our kconfig, so probably not an option here.